pkg/plugins/manager/installer.go

```diff
@@ -134,7 +134,7 @@ func (m *PluginInstaller) Remove(ctx context.Context, pluginID string) error {
 		return plugins.ErrPluginNotInstalled
 	}
 
-	if !plugin.IsExternalPlugin() {
+	if plugin.IsCorePlugin() || plugin.IsBundledPlugin() {
```

Comment:
Note: This is changed to be more explicit in case a new plugin class is introduced where we don't want this error to be returned